<launch>
  <node name="factory_floor_manager" pkg="factory_floor_manager" type="factory_floor_manager_node" args="--noui"/>
</launch>
